api/lib/router/routes/scanner/scanner.ts

```diff
@@ -189,14 +264,14 @@ export class AdminScannerController extends ScannerController implements IExpres
   }
 
   /**
-   * @api {get} /admin/scanner/registrations Obtain all registrations
+   * @api {get} /scanner/registrations Obtain all registrations
    * @apiVersion 2.0.0
    * @apiName Obtain all registrations (Scanner)
-   *
    * @apiGroup Admin Scanner
    * @apiPermission TeamMemberPermission
    * @apiPermission ScannerPermission
    * @apiUse AuthArgumentRequired
+   * @apiUse ApiKeyArgumentRequired
    * @apiSuccess {Registration[]} Array of current registrations
```

Comment:
Please be more specific about the data format I'll get back. Or at least make it clear what each registration data looks like.

Reply:
Can you add this to the backlog? The change requested here is large enough to be its own PR